frontend/src/components/ShoppingList.tsx

```diff
@@ -41,17 +39,66 @@ export default function ShoppingList() {
         return <p>Loading...</p>
     }
 
+    function deleteList(id:string) {
+
+        axios.delete("/api/shop/" + id)
```

Comment:
Du verwendest Mal einzelne und mal doppelte Anführungsstriche, es wichtig immer konsistent zu bleiben. Hier kannst du Backticks verwenden:
axios.delete(`/api/shop/${id}`)
Das ist der moderne Ansatz, wenn wir Variablen in Strings einfügen möchten.

Reply:
Lass uns hierzu bitte auch nochmal sprechen, nach hinzufügen der neuen Schreibweise, funktioniert der Button nicht mehr.